read-the-docs/source/Usage.rst

```diff
@@ -100,6 +100,7 @@ URL of the ticket. An example is below.
 
 .. note::
 
-    For ServiceNow, Jira, Bugzilla and Redmine the user-accessible methods return a ``ticket_content``
-    field, which contains a json representation of the current ticket's content.
+    For all ticketing tools the user-accessible methods return a ``ticket_content``
+    field, which contains a json representation of the current ticket's content with
+    exception of RT returning list of strings.
```

Comment:
After the new changes, you can remove the "with exception of RT..." part.

Reply:
Removed. Thanks for being so attentive to details like this. I tend to forget about them.